src/relay/pass/fuse_ops.cc

```diff
@@ -747,6 +737,30 @@ class FuseMutator : private ExprMutator {
       return ExprMutator::VisitExpr_(call);
     }
   }
+
+  Expr VisitExpr_(const TupleNode* tuple) {
+    auto* ret_group = gmap_.at(tuple)->FindRoot();
```

Comment:
We will need to check if tuple is at the root of the group and handle the case. opt_level=0 might review this problem